features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -668,3 +668,12 @@ CREATE TABLE IF NOT EXISTS  IDN_SAML2_ASSERTION_STORE (
   PRIMARY KEY (ID)
 );
 
+CREATE TABLE IF NOT EXISTS  IDP_OIDC_PROPERTY (
+  ID INTEGER NOT NULL AUTO_INCREMENT,
+  TENANT_ID  INTEGER,
+  CONSUMER_KEY  VARCHAR(255) ,
+  PROPERTY_KEY  VARCHAR(255) NOT NULL,
+  PROPERTY_VALUE  VARCHAR(2047) ,
+  PRIMARY KEY (ID),
+  FOREIGN KEY (CONSUMER_KEY) REFERENCES IDN_OAUTH_CONSUMER_APPS(CONSUMER_KEY) ON DELETE CASCADE
+);
```

Comment:
Add new line to the end of file, do the same at other places

Reply:
Fixed